health/health.d/wmi.conf

```
@@ -120,14 +120,52 @@ component: Network
        to: sysadmin
 
 
+## Service
+
+ template: wmi_service_status
+       on: wmi.service_status
+    class: Errors
+     type: Windows
+component: Services
+       os: *
+    hosts: *
+ families: *
+   lookup: max -60s unaligned of error,stressed,nonrecover
+    units: failures
+    every: 1m
+     warn: $this >= 1
+    delay: down 10m multiplier 1.5 max 30m
+     info: number of errors reported for $label:service in the last 10 minutes
```

```suggestion
     info: service $label:service is not operating properly
```

Or something generic like this that covers all problematic statuses.